### PR TITLE
test: Optimize E2E by removing redundant steps

### PR DESCRIPTION
**Pull request by novakzaballa**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes

- 

## How did you test this code?

- `npm run test`

### CODE REVIEW

File: frontend/e2e/tests/segment-test.ts [hunk 1]
```diff
@@ -32,18 +32,8 @@ export const testSegment1 = async () => {
   log('Login')
   await login(E2E_USER, PASSWORD)
 
-  log('Create Organisation')
-  await click(byId('create-organisation-btn'))
-  await setText('[name="orgName"]', 'Bullet Train Ltd 2')
-  await click('#create-org-btn')
-  await waitForElementVisible(byId('project-select-page'))
+  await click('#project-select-1')
```
Comment (kyle-ssg) [inline frontend/e2e/tests/segment-test.ts:35]: If these are all removed, is create project getting end to end tested anywhere?

**Review by kyle-ssg — APPROVED** [review]

File: api/e2etests/e2e_seed_data.py [hunk 2]
```diff
@@ -34,12 +35,47 @@ def seed_data() -> None:
...
+        identifier=settings.E2E_IDENTITY, environment=project2_dev_env
+    )
+
+    Identity.objects.create(
+        identifier=settings.E2E_IDENTITY, environment=project3_dev_env
+    )
+
+    Identity.objects.create(
+        identifier=settings.E2E_IDENTITY, environment=project4_dev_env
+    )
```
Comment (matthewelwell) [inline api/e2etests/e2e_seed_data.py]: Feels like this a lot of this could be done in a for loop?

**Review by zachaysan — APPROVED** [review]

File: api/e2etests/e2e_seed_data.py (hunk 2 above)
Comment (novakzaballa) [reply api/e2etests/e2e_seed_data.py]: You're right, Done.

File: api/e2etests/e2e_seed_data.py [hunk 3]
```diff
@@ -34,11 +35,41 @@ def seed_data() -> None:
...
+        for env_name in project_info["environments"]:
+            environment = Environment.objects.create(name=env_name, project=project)
+            environments.append(environment)
+
+    # Create identities
+    identities_test_data = [
+        {"identifier": settings.E2E_IDENTITY, "environment": environments[2]},
+        {"identifier": settings.E2E_IDENTITY, "environment": environments[3]},
+        {"identifier": settings.E2E_IDENTITY, "environment": environments[4]},
+    ]
```
Comment (matthewelwell) [inline api/e2etests/e2e_seed_data.py:72]: We should add a comment explaining why we're only creating identities for 3 of the 5 environments.

File: api/e2etests/e2e_seed_data.py [hunk 4]
```diff
@@ -34,11 +35,41 @@ def seed_data() -> None:
...
+            "name": "My Test Project",
+            "environments": [
+                "Development",
+                "Production",
+            ],
+        },
+        {"name": "My Test Project 2", "environments": ["Development"]},
+        {"name": "My Test Project 3", "environments": ["Development"]},
+        {"name": "My Test Project 4", "environments": ["Development"]},
+    ]
```
Comment (matthewelwell) [inline api/e2etests/e2e_seed_data.py:50]: Can we add a comment explaining why different projects have different environments?

Comment (novakzaballa) [reply api/e2etests/e2e_seed_data.py:50]: Added.

File: api/e2etests/e2e_seed_data.py (hunk 3 above)
Comment (novakzaballa) [reply api/e2etests/e2e_seed_data.py:72]: Added.

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**